tests/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.Tests.ps1

```diff
@@ -114,6 +114,25 @@ Describe 'MousePointer' {
     }
 }
 
+Describe 'DynamicScrollbar'{
+
+   It 'Sets desired value.'{
+      $testingState = @{ Show = $false }
+      Invoke-DscResource -Name Scrollbar -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $testingState
+
+      # Randomly generate desired state. Minimum is set to 1 to avoid using KeepCurrentValue
+      $desiredScrollbarBehavior = [Scrollbar](Get-Random -Maximum 2 -Minimum 1)
```

Comment:
You don't need to randomly generate the desired state. You can just set it to $true and verify that it gets set correctly.

Make sure to also invoke the 'Test' method and verify that it is correct.

Reply:
Why not? What's so special about either this resource, or the other resources, that it's needed there but not here? 

This suggestion contradicts a [previous](https://github.com/microsoft/winget-dsc/pull/49#discussion_r1723890031) suggestion.

Reply:
It is not necessary to have a 'keepcurrentvalue' state for a binary setting. Therefore you can just remove the random number generator and simply check if the state changes after setting it from false -> true.